input/pagecontent/changes.md

```diff
@@ -39,16 +38,28 @@ To help implementers, only the more significant changes are listed here.
 <ul>
 <li><a href="SearchParameter-indigenous-status.html">AustralianIndigenousStatus</a></li>
 <li><a href="SearchParameter-encounter-discharge-disposition.html">EncounterDischargeDisposition</a></li>
-<li><a href="SearchParameter-patient-gender-identity.html">PatientGenderIdentity</a></li>
+<li><a href="SearchParameter-gender-identity.html">GenderIdentity</a> (<a href="https://jira.hl7.org/browse/FHIR-43718">FHIR-43718</a>)</li>
 </ul>
 </li>
 <li>Changed Patient.extension in <a href="StructureDefinition-au-patient.html">AU Base Patient</a>: 
-<ul><li>to add Individual Pronouns and Person Recorded Sex Or Gender</li>
+<ul><li>to add Individual Pronouns (<a href="https://jira.hl7.org/browse/FHIR-43719">FHIR-43719</a>) and Person Recorded Sex Or Gender</li>
 <li>to add Australian Veteran Status</li>
-<li>to remove the value slice with the binding to Gender Identity Response (extensible) from genderIdentity (<a href="https://github.com/hl7au/au-fhir-base/issues/815">au-fhir-base #815</a>)</li>
+<li>to remove genderIdentity extension and replace it with the Individual Gender Identity extension (<a href="https://jira.hl7.org/browse/FHIR-43718">FHIR-43718</a>)</li>
 </ul>
 <li>Changed context of extension <a href="StructureDefinition-indigenous-status.html">Australian Indigenous Status</a> to add Person and RelatedPerson.</li>
-<li>Added new invariant to Patient root element for genderIdentity extension terminology (<a href="https://github.com/hl7au/au-fhir-base/issues/815">au-fhir-base #815</a>).</li></li> 
+<li>Removed genderIdentity terminology constraint and replaced it with the Australian Individual Gender Identity terminology constraint in AU Base Patient (<a href="https://jira.hl7.org/browse/FHIR-43718">FHIR-43718</a>).</li>
```

Comment:
Don't say removed and then added for changes all happening in the same version. This change log isn't against last snapshot it is against the last official release.